tests/manage/pv_services/pvc_clone/test_clone_when_pvc_full.py

```diff
@@ -91,6 +95,34 @@ def test_clone_when_full(self, pvc_clone_factory, pod_factory):
         cloned_pvcs = [pvc_clone_factory(pvc_obj) for pvc_obj in self.pvcs]
         log.info("Created clone of the PVCs. Cloned PVCs are Bound")
 
+        # Bug 2042318
+        for clone_pvc in cloned_pvcs:
+            if (
+                clone_pvc.get().get("spec").get("storageClassName")
+                == "ocs-storagecluster-cephfs"
+            ):
+                pv = clone_pvc.get().get("spec").get("volumeName")
+                func_calls = "clone"
+                error_msg = f"{pv} failed to create clone from subvolume"
+                csi_cephfsplugin_pod_objs = res_pod.get_all_pods(
+                    namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
+                    selector=["csi-cephfsplugin-provisioner"],
+                )
+            relevant_pod_logs = None
+            for pod_obj in csi_cephfsplugin_pod_objs:
+                pod_log = res_pod.get_pod_logs(
+                    pod_name=pod_obj.name, container="csi-cephfsplugin"
+                )
+
+                if func_calls in pod_log:
```

Comment:
What does this line check ?
pod_log will be a string. There are 2 provisioner pods. Only one of them will be active at a time. I think checking a match string "clone" in the entire log isn't helpful .